.eslintrc.json

```diff
@@ -8,6 +8,7 @@
   "extends": "airbnb",
   "parser": "babel-eslint",
   "rules": {
-    "camelcase": 0
+    "camelcase": 0,
+    "object-shorthand": [2, "properties"]
```

Comment:
eslint insisted I would use [object-shorthand](http://eslint.org/docs/rules/object-shorthand), but then our babel setup didn’t actually support that. After trying to find the right preset/plugin for about 10 min I gave up.